### DISCUSSION TITLE
Is there a way when the group user upload the package is only visible by the group user by using api

### DISCUSSION THREAD

**Discussion by alan7622** [fossology/fossology#2024] [Q&A]
searching for long time but didn't find API that have this access, thanks
![image](https://user-images.githubusercontent.com/48542173/122535282-fb30a980-d055-11eb-8c94-ea4a46e73358.png)

**Comment by GMishx** [comment]
Hey, sorry for coming so late to this. Actually there is a parameter to do that. In all POST requests to `/uploads`, you can set the `public` header to either `private`, `protected` or `public` corresponding to the option 5 in screenshot (in same order). For your case, it should be `private`.
https://github.com/fossology/fossology/blob/a3ba2c87f5dd7f39ff4b6392ea8956cbfd099e27/src/www/ui/api/documentation/openapi.yaml#L395-L405

Also, there is a `groupName` header which allows you to change the group for each request. Hope this information will be helpful.
